mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -181,6 +184,35 @@ void registerGroupShouldCreateRetryExchange() throws Exception {
     class ConcurrentTest implements EventBusConcurrentTestContract.MultiEventBusConcurrentContract,
         EventBusConcurrentTestContract.SingleEventBusConcurrentContract {
 
+        @Test
+        void rabbitMQEventBusCannotHandleHugeDispatchingOperations() throws Exception {
+            EventBusTestFixture.MailboxListenerCountingSuccessfulExecution countingListener1 = newCountingListener();
+
+            eventBus().register(countingListener1, new EventBusTestFixture.GroupA());
+            int totalGlobalRegistrations = 1; // GroupA + GroupB + GroupC
+
+            int threadCount = 10;
+            int operationCount = 10000;
+            int totalDispatchOperations = threadCount * operationCount;
+            eventBus = (RabbitMQEventBus) eventBus();
```

Comment:
Using the `eventBus` class field directly returned by `eventBus()`, this cast can be avoided...